engine/caches.go

```diff
@@ -443,6 +443,22 @@ func (chS *CacheS) V1GetCacheStats(ctx *context.Context, args *utils.AttrCacheID
 	return
 }
 
+type CacheStatsWithMetadata struct {
+	CacheStatistics map[string]*ltcache.CacheStats
```

Comment:
Have the inclination towards Stats here instead of CacheStatistics. Maybe we need to discuss about it.